### ISSUE TITLE
File rules documentation config snippet incorrect

### ISSUE THREAD

**Issue by dyscalculia94** [AcademySoftwareFoundation/OpenColorIO#1673]
Hi all,

I've been reading about file rules and decided to modify the ACES 1.2 config to include the file rules from OCIO documentation:
```
files_rules:
  - !<Rule> {name: LogC, extension: "*", pattern: "*LogC*", colorspace: ARRI LogC}
  - !<Rule> {name: OpenEXR, extension: "exr", pattern: "*", colorspace: ACEScg}
  - !<Rule> {name: TIFF, regex: ".*\.TIF?F$", colorspace: sRGB}
  - !<Rule> {name: ColorSpaceNamePathSearch}
  - !<Rule> {name: Default, colorspace: default}
```
I've also changed the config version to v2, since that config is v1.

However, after using `ociocheck` on that config I've discovered that there are two problems with it:
1. Typo: `files_rules` instead of `file_rules`
2. Error because of the unknown escape character : `.` This one I fixed by deleting the backslash (`\`) from the `TIFF` rule.

I was thinking of submitting a pull request to fix this. Is this something that should be fixed via pull request or is it something that OCIO devs would fix internally, since it's a really small and simple fix?

**Comment by zachlewis** [comment]
> * Error because of the unknown escape character : `.` This one I fixed by deleting the backslash (`\`) from the `TIFF` rule

If the intention is to represent a literal `.` dot before the extension,  I think we might actually need to add a second backslash  to escape YAML's own parsing of the backslash character: `".*\\.TIF?F$"`

**Comment by dyscalculia94** [comment]
Yes, you're probably right. That's probably supposed to match files with the `.TIF` extension, as well as other similar to that.

**Comment by doug-walker** [comment]
Closing as fixed by PR #1738.
